files/acme/policy/clusterrolebinding.yml

```diff
@@ -0,0 +1,12 @@
+apiVersion: v1
+kind: ClusterRoleBinding
+metadata:
+  name: acme-controller
+roleRef:
+  name: acme-controller
+subjects:
+- kind: ServiceAccount
+  name: acme
+  namespace: acme
```

Comment:
this locks it in on always being deployed as `acme` ... while the above template allows for setting the namespace to something else than `acme`. Can we change this file to a template as well to allow for a custom name?